web/js/data/results.js

```diff
@@ -1,45 +1,59 @@
-/**
- * @module wv.data
- */
-var wv = wv || {};
-wv.data = wv.data || {};
-
-wv.data.results = wv.data.results || {};
-
-wv.data.results.antiMeridianMulti = function (maxDistance) {
+import $ from 'jquery';
+
+import OlGeomMultiPolygon from 'ol/geom/multipolygon';
+import OlGeomMultiPoint from 'ol/geom/multipoint';
+import OlFormatGeoJSON from 'ol/format/geojson';
+import OlGeomPolygon from 'ol/geom/polygon';
+import OlGeomPoint from 'ol/geom/point';
+import olExtent from 'ol/extent';
+import loClone from 'lodash/clone';
+import loEach from 'lodash/each';
+import {dataCmrRoundTime, dataCmrGeometry} from './cmr';
+import util from '../util/util';
+import {dataHelper} from '../edsc';
+import {
+  CRS_WGS_84,
+  mapIsPolygonValid,
```

Comment:
What's the difference between prefixing all of these with `map` and prefixing all of the Worldview JavaScript files with `wv.`? Does the extra prefix provide some value? Or are we being more verbose than we need to be?

Reply:
I'd say we essentially got rid of the`wv` in `wv.map.something` and are left with the `map.something` as camel case. The prefix stands for the folder in which something is in. We can't just call a method `parser`, we need `dataParser` if we want to export unique names across the board. `mapIsPolygonValid` would be fine as `isPolygonValid` but I'm just trying to stick with a convention.